app/services/certificate_store.rb

```diff
@@ -144,4 +145,16 @@ def extract_certs(raw)
       Certificate.new(OpenSSL::X509::Certificate.new(pem + END_CERTIFICATE)) if pem.strip.present?
     end.compact.select(&:ca_capable?)
   end
+
+  def alert_on_expired_cert(cert)
+    now = Time.zone.now
+    return if cert.not_after >= now
+
+    NewRelic::Agent.notice_error("\
+Certificate Expired. \
+Expiration: #{cert.not_after}, \
+Subject: #{cert.subject}, \
+Issuer: #{cert.issuer}, \
+Key ID: #{cert.key_id}")
```

Comment:
The trailing slashes are hard to read, what if we use ActiveSupport's `String#squish`?

```suggestion
    NewRelic::Agent.notice_error(<<-STR.squish)
      Certificate Expired.
      Expiration: #{cert.not_after},
      Subject: #{cert.subject},
      Issuer: #{cert.issuer},
      Key ID: #{cert.key_id}
    STR
```

Reply:
nice!